rapid7_insightappsec/help.md

```diff
@@ -368,10 +462,19 @@ This action is used to submit a new scan action.
 
 ##### Input
 
-|Name|Type|Default|Required|Description|Enum|
-|----|----|-------|--------|-----------|----|
-|scan_id|string|None|True|Scan UUID|None|
-|action|string|PAUSE|True|Action to take|['PAUSE', 'RESUME', 'STOP', 'CANCEL']|
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|action|string|Pause|True|Action to take|['Pause', 'Resume', 'Stop', 'Cancel']|Pause|
+|scan_id|string|None|True|Scan UUID|None|008eaffe-90ce-4de9-8601-40414391c21c|
+
+Example input:
+
+```
+{
+  "action": "PAUSE",
```

Comment:
Is this value case sensitive? If, yes then we need to match it to table `Example` value.